### ISSUE TITLE
sound: research and create a demo with dialogue prior to integrating text to speech

### ISSUE THREAD

**Issue by sangerr** [uchicago-cs/chiventure#1579]
In this issue, we will be creating a demo of an instance in the game where there will be dialogue between two npc's or a player and an npc. In order to create this demo, we will need to research and develop an understanding of the structs and implementation necessary to have a game dialogue. After we have created a demo with dialogue, we will create a new issue where we can integrate the text to speech into the demo and have it run as the game progresses.

Assignees: Rhea, Victor, Jake

**Comment by sangerr** [comment]
To initially research how to create a demo with dialogue, I have been looking at the src/npc/examples/dialogue_example.c file, as that team created a detailed dialogue in the game example. For our demo, we will need to have a more concise dialogue between players, as well as figure out at what nodes we want sound from text to speech.

**Comment by sangerr** [comment]
Closing Statement: This issue is now closed as we implemented an example game with a conversation between an npc and player, which was merged into dev, and now are ready to work on the issue of integrating the text to speech into the example. While working on this, Victor and Jake spent time researching the existing npc structs and examples, and developed an understanding of how to orchestrate a dialogue between a character and npc. Rhea discussed these findings with Victor and Jake and translated that into writing the implementation for the example game, in collaboration Victor and Jake (working from Rhea's computer while in the same room).
